src/helm/benchmark/static/benchmarking.js

```diff
@@ -385,6 +363,71 @@ $(function () {
     return JSON.stringify([displayRequest.instance_id, displayRequest.perturbation || 'original']);
   }
 
+
+  Handlebars.registerHelper('highlightNewWords', function (perturbed, unperturbed) {
```

Comment:
For consistency, should we use the `(args) =>` notation?

Reply:
Changed to arrow notation.